### PR TITLE
Supply multiple beacon node endpoints.

### LINKED ISSUE #384: Specify multiple beacon nodes for a validator client
See: https://github.com/ConsenSys/teku/pull/6001

https://hackmd.io/9UKp7Mc8QQKBBmDkeUjRnQ?view

### PR DESCRIPTION
**Pull request by bgravenorst**
Signed-off-by: bgravenorst <byron.gravenorst@consensys.net>

## Describe the change

We can now configure multiple beacon nodes for a validator client.

## Issue fixed

Fixes #384 

## Preview

- https://pegasys-teku--386.com.readthedocs.build/en/386/Reference/CLI/Subcommands/Validator-Client/#beacon-node-api-endpoint-beacon-node-api-endpoints
- https://pegasys-teku--386.com.readthedocs.build/en/386/HowTo/Get-Started/Run-Teku/#start-the-validator

### CODE REVIEW

File: docs/Reference/CLI/Subcommands/Validator-Client.md [hunk 1]
```diff
@@ -6,33 +6,39 @@ title: Validator client subcommand options
...
     ```bash
-    teku vc --beacon-node-api-endpoint=http://192.138.10.12
+    teku vc --beacon-node-api-endpoint=http://192.138.10.12:5051,http://192.140.11.44:5051
     ```
 
 === "Environment variable"
 
     ```bash
-    TEKU_BEACON_NODE_ENDPOINT=http://192.138.10.12
+    TEKU_BEACON_NODE_ENDPOINT=http://192.138.10.12,http://192.140.11.44:5051
```
Comment (StefanBratanov) [inline docs/Reference/CLI/Subcommands/Validator-Client.md:26]: Should it not be TEKU_BEACON_NODE_API_ENDPOINT?

**Review by StefanBratanov — CHANGES_REQUESTED** [review]
LGTM

File: docs/HowTo/Get-Started/Run-Teku.md [hunk 2]
```diff
@@ -94,10 +94,15 @@ a Teku as a validator.
...
-Specify the beacon node using the
-[`--beacon-node-api-endpoint`](../../Reference/CLI/Subcommands/Validator-Client.md#beacon-node-api-endpoint)
+Specify one or more beacon nodes using the
+[`--beacon-node-api-endpoint`](../../Reference/CLI/Subcommands/Validator-Client.md#beacon-node-api-endpoint-beacon-node-api-endpoints)
 option.
 
+!!! important
+
+    You can supply multiple beacon node endpoints to the validator, the first one will be used as the
+    primary node, and others as failovers.
```
Comment (alexandratran) [inline docs/HowTo/Get-Started/Run-Teku.md:104]:
```suggestion
    You can supply multiple beacon node endpoints to the validator. The first one is used as the
    primary node, and others as failovers.
```

File: docs/Reference/CLI/Subcommands/Validator-Client.md [hunk 3]
```diff
@@ -6,33 +6,39 @@ title: Validator client subcommand options
...
     ```bash
-    beacon-node-api-endpoint: "http://192.138.10.12"
+    beacon-node-api-endpoint: ["http://192.138.10.12","http://192.140.11.44:5051"]
     ```
 
-Endpoint of the beacon node's REST API. The default is `http://127.0.0.1:5051`.
+Endpoint of the beacon node's REST API. You can configure multiple beacon nodes by providing a
+comma-separated list of beacon node API endpoints.
+
+If multiple beacon node endpoints are configured, the first one will be used as primary and others
```
Comment (alexandratran) [inline docs/Reference/CLI/Subcommands/Validator-Client.md:38]:
```suggestion
If multiple beacon node endpoints are configured, the first one is used as primary and others
```

**Review by alexandratran — APPROVED** [review]
Minor suggestion. LGTM.

**Review by StefanBratanov — APPROVED** [review]
LGTM

**Pull request merged**